package.json

```diff
@@ -5,28 +5,29 @@
   "version": "1.0.11",
   "license": "MIT",
   "scripts": {
-    "test": "jest --env=jsdom --browser",
+    "test": "jest --env=node",
     "start": "ts-node src/index.ts",
-    "build": "tsc -p tsconfig.json --outDir lib",
+    "install": "tsc -p tsconfig.json --outDir lib",
```

Comment:
I would revert this back to build. It breaks the prepublish hook as-is, and generally is better described as a build step.